### PR TITLE
fix(user-group): use `slapd` `user` and `group` from `map.jinja`

### PR DESCRIPTION
**Pull request by sticky-note**
Use slapd_user and slapd_group from `map.jinja` for `slapd.conf` owner and group owner

### CODE REVIEW

File: openldap/server.sls [hunk 1]
```diff
@@ -7,8 +7,8 @@
   file.managed:
     - source: salt://openldap/files/slapd.conf
     - template: jinja
-    - user: root
-    - group: {{ openldap.su_group }}
+    - user: {{ openldap.slapd_user }}
+    - group: {{ openldap.slapd_group }}
```
Comment (myii) [inline openldap/server.sls]: Just confirming, is this change only needed here?  The other places where this occurs:

https://github.com/saltstack-formulas/openldap-formula/blob/9f758a4a599bb521b2b40cc9b4eab249c53bb4ec/openldap/server.sls#L19-L24

https://github.com/saltstack-formulas/openldap-formula/blob/9f758a4a599bb521b2b40cc9b4eab249c53bb4ec/openldap/server.sls#L35-L38

https://github.com/saltstack-formulas/openldap-formula/blob/ae96c9f1e301e3bd7d10fb7538c9a18113ab28a7/openldap/client.sls#L2-L10

Comment (sticky-note) [reply openldap/server.sls]: `default` files are generally owned by `root`.
On the `client` conf . it is not necessary neither

File: pillar.example [hunk 2]
```diff
@@ -4,8 +4,8 @@ openldap:
     base: 'dc=example,dc=com'
     uri: ldap://ldap.example.com
     slapd_conf: '/etc/ldap/slapd.conf' # if no dynamic configuration on Debian
-    user: 'openldap'
-    group: 'openldap'
+    slapd_user: 'openldap'
+    slapd_group: 'openldap'
```
[the remark this answers is not in the document]
Comment (sticky-note) [reply pillar.example]: Not necessary but clearer

**Comment by sticky-note** [comment]
@myii Changes done.

**Comment by myii** [comment]
Merged, thanks @sticky-note.

**Pull request merged**